File: docs/evidence/2026-09-25-rust-core-spikes/swig/ruby/t.rb
require_relative "aprv"
b = File.read("../../../../../fixtures/public-receipts/receipt-sandbox-g5.b64").gsub(/\s/, "")
puts "ruby: " + Aprv.verify_receipt_base64(Aprv.aprv_verifier_new_receipt("dev.bonzer.weeka.app"), b)[0,40]
begin; Aprv.verify_receipt_base64(Aprv.aprv_verifier_new_receipt("x.y"), b); rescue ArgumentError => e; puts "ruby error ok: #{e.message}"; end

File: docs/evidence/2026-09-25-rust-core-spikes/swig/perl/t.pl
use lib "."; use aprv;
open(my $f, "<", "../../../../../fixtures/public-receipts/receipt-sandbox-g5.b64"); local $/; my $b = <$f>; $b =~ s/\s//g;
print "perl: ", substr(aprv::verify_receipt_base64(aprv::aprv_verifier_new_receipt("dev.bonzer.weeka.app"), $b), 0, 40), "\n";
eval { aprv::verify_receipt_base64(aprv::aprv_verifier_new_receipt("x.y"), $b) }; print "perl error ok: $@" if $@;
